e2e/test/scenarios/admin/datamodel/metadata.cy.spec.js

```diff
@@ -148,6 +149,76 @@ describe("scenarios > admin > datamodel > metadata", () => {
     openOptionsForSection("Display values");
     popover().findByText("Custom mapping");
   });
+
+  describe("column formatting options", () => {
+    beforeEach(() => {
+      cy.intercept("PUT", "/api/field/*").as("updateField");
+      cy.intercept("GET", "/api/field/*").as("getField");
+    });
+
+    it("should only show currency formatting options for currency fields", () => {
+      cy.visit(
+        `/admin/datamodel/database/${SAMPLE_DB_ID}/schema/${SAMPLE_DB_SCHEMA_ID}/table/${ORDERS_ID}/field/${ORDERS.DISCOUNT}/formatting`,
+      );
+
+      cy.wait("@getField");
+
+      cy.findByTestId("column-settings").within(() => {
+        cy.findByText("Unit of currency");
+        cy.findByText("Currency label style");
+      });
+
+      cy.visit(
+        `/admin/datamodel/database/${SAMPLE_DB_ID}/schema/${SAMPLE_DB_SCHEMA_ID}/table/${ORDERS_ID}/field/${ORDERS.QUANTITY}/formatting`,
+      );
+
+      cy.wait("@getField");
+
+      cy.findByTestId("column-settings").within(() => {
+        // shouldnt show currency settings by default for quantity field
+        cy.findByText("Unit of currency").should("not.exist");
+        cy.findByText("Currency label style").should("not.exist");
```

Comment:
The settings are only being hidden - `display: none;` - so that is why these tests are failing
```suggestion
        cy.findByText("Unit of currency").should("not.be.visible");
        cy.findByText("Currency label style").should("not.be.visible");
```